docs/source/get_started/kedro_concepts.md

```diff
@@ -99,10 +132,10 @@ Use the `local` subfolder for **settings that should not be shared**, such as ac
 
 By default, Kedro creates one file, `credentials.yml`, in `conf/local`.
 
-### `data`
-
-The `data` folder contains multiple subfolders to store project data. We recommend you put raw data into `raw` and move processed data to other subfolders according to the [commonly accepted data engineering convention](https://towardsdatascience.com/the-importance-of-layered-thinking-in-data-engineering-a09f685edc71).
-
 ### `src`
 
 This subfolder contains the project's source code.
+
+### Customising your project structure
+
+While the default Kedro structure is recommended for collaboration and standardisation, it is possible to adapt the folder structure if necessary. This flexibility allows you to tailor the project to your needs while maintaining a consistent and recognisable structure.
```

Comment:
Maybe you can complement this sentence by adding a bit more information or what _must_ be there, for example https://github.com/kedro-org/kedro/pull/3852/files#diff-0b95e768cfe29967b90e84676a5943c8da00182809104e1de48f78534db61d66R17-R27